tests/util/test_dataframe_ops.py

```diff
@@ -52,3 +61,240 @@ def test_parse_check(mock_parse_probtest_csv, setup_csv_files):
 
     pd.testing.assert_frame_equal(df_ref, expected_ref)
     pd.testing.assert_frame_equal(df_cur, expected_cur)
+
+
+def test_force_monotonic():
+    """
+    Test that the function modify the dataframe forcing the values of every line
+    to become non-decreasing monotonic along the columns
+    """
+    # Creation of a DataFrame with MultiIndex on the columns
+    arrays = [
+        ["var1", "var1", "var2", "var2"],
+        ["mean", "max", "mean", "max"],
+    ]
+    columns = pd.MultiIndex.from_arrays(arrays)
+    data = [
+        [1, 5, 2, 7],
+        [3, 2, 1, 9],
+        [2, 8, 5, 4],
+    ]
+    df = pd.DataFrame(data, columns=columns)
+
+    force_monotonic(df)
+
+    # Property verification
+    for stat in df.columns.levels[1]:
+        sub_df = df.loc[:, (slice(None), stat)]
+        assert (sub_df.diff(axis=1).fillna(0) >= 0).all().all()
+
+    # Comparison with expected dataframe
+    expected = pd.DataFrame([[1, 5, 2, 7], [3, 2, 3, 9], [2, 8, 5, 8]], columns=columns)
+    pd.testing.assert_frame_equal(df, expected)
+
+
+def test_compute_rel_diff_basic():
+    """
+    Test that the function is giving the expected values with normal numbers
```

Comment:
I would avoid to use normal numbers, and use basic numbers in this case
```suggestion
    Test that the function is giving the expected values with basic numbers
```
This also applies to a test below.